sourced/ml/extractors/type_children.py

```diff
@@ -0,0 +1,122 @@
+import numpy
+import re
+from collections import defaultdict
+
+from sourced.ml.algorithms import UastNodes2Bag
+from sourced.ml.extractors import BagsExtractor, register_extractor, get_names_from_kwargs,\
+    filter_kwargs
+
+
+@register_extractor
+class TypeChildrenBagExtractor(BagsExtractor):
+    NAME = "typechildren"
+    NAMESPACE = "tc."
+    OPTS = dict(get_names_from_kwargs(UastNodes2Bag.__init__))
+    OPTS = BagsExtractor.OPTS.copy()
```

Comment:
I believe that it should be `OPTS.update(BagsExtractor.OPTS)` or previous line should be removed